<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatingEvents</class>
 <widget class="QDialog" name="CreatingEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="TitleLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Выбрать участников события</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="DescriptionTextEdit"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QListWidget" name="ParticipantslistWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Место провидения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="LocationLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="TypeLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Приоритет</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="PriorityLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Дата начала</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDateTimeEdit" name="StartDateTimeEdit">
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>5</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Дата окончания</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateTimeEdit" name="EndDateTimeEdit">
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>5</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="SaveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
